hermeto/core/utils.py

```diff
@@ -136,7 +136,8 @@ def _fast_copy(src: Path, dest: Path, *, follow_symlinks: bool = True) -> int:
             raise _FastCopyFailedFallback()
 
         try:
-            while nbytes := os.copy_file_range(srcfd, destfd, count=_get_blocksize(srcfd)):
+            # mypy: `os` module has no attribute "copy_file_range" on some platforms
```

Comment:
> The fast copy function relies on the Linux-specific os.copy_file_range
> syscall. It should not be used on other platforms as it would fail. Now
> we explicitly raise an exception on non-Linux systems to trigger the
> fallback to regular copy methods.
The commit msg needs an update as it currently doesn't match the  actual changes proposed.